_sass/components/popovers/popover-search.scss

```diff
@@ -0,0 +1,72 @@
+.popover-search {
+  border: 0;
+  bottom: 0;
+  height: auto;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  position: relative;
+
+  &:not(.lunr-hidden) {
+    background: #fff;
+    max-height: 349px;
+    transition: all .10s ease-in-out;
```

Comment:
Let's be specific about what is transitioning here, and .10s can be shorted to .1s